src/client/components/financial-assessment/DeductibleExpensesSection.jsx

```diff
@@ -4,7 +4,7 @@ import { connect } from 'react-redux';
 import ErrorableCheckbox from '../form-elements/ErrorableCheckbox';
 import ErrorableTextInput from '../form-elements/ErrorableTextInput';
 import { isValidField, isValidMonetaryValue } from '../../utils/validations';
-import { updateReviewStatus, veteranUpdateField } from '../../actions';
+import { veteranUpdateField } from '../../actions';
```

Comment:
I think there are more things that need to be removed in this file, the `editButton` stuff.